tests/unit/test__main.py

```diff
@@ -35,47 +35,115 @@
     "verbose": True,
 }
 
-CONFIG_RUNNER_ARGS_1 = {
+BASE_RUNNER_ARGS = {
     "verbose": False,
     "log_level": "INFO",
+    "dry_run": False,
     "command": "configs",
     "validation_config_cmd": "run",
-    "dry_run": False,
+    "kube_completions": True,
+}
+CONFIG_RUNNER_ARGS_1 = BASE_RUNNER_ARGS | {
     "config_file": "gs://pso-kokoro-resources/resources/test/unit/test__main/3validations/first.yaml",
     "config_dir": None,
-    "kube_completions": True,
 }
-CONFIG_RUNNER_ARGS_2 = {
+CONFIG_RUNNER_ARGS_2 = BASE_RUNNER_ARGS | {
+    "config_dir": "gs://pso-kokoro-resources/resources/test/unit/test__main/3validations",
+}
+CONFIG_RUNNER_ARGS_3 = BASE_RUNNER_ARGS | {
+    "config_dir": "gs://pso-kokoro-resources/resources/test/unit/test__main/4partitions",
+}
+CONFIG_RUNNER_ARGS_4 = CONFIG_RUNNER_ARGS_3 | {
```

Comment:
Ive made sure not to change the values of `CONFIG_RUNNER_ARGS_1` + `CONFIG_RUNNER_ARGS_2` + `CONFIG_RUNNER_ARGS_3` + `CONFIG_RUNNER_ARGS_4`, Ive used merge syntax to reduce repeated rows.

(I can revert this if its preferred)

Reply:
Reverted here -> https://github.com/GoogleCloudPlatform/professional-services-data-validator/pull/1329/commits/15b09907da4942de06aa5e3dc7475f1d5f199bc6